loginCore/src/main/java/com/hamdan/forzenbook/login/core/domain/usecase/LoginEntrys.kt

```diff
@@ -0,0 +1,9 @@
+package com.hamdan.forzenbook.login.core.domain.usecase
+
+import com.hamdan.forzenbook.core.Entry
+import com.hamdan.forzenbook.core.LoginError
+
+data class LoginEntrys(
+    val email: Entry = Entry("", LoginError.EmailError.Length),
```

Comment:
I don't think it makes sense for the default to be a specific type like that. Maybe make it nullable or add a `.None`

Reply:
I'll add a .None, the .Empty I wanted to have a while ago was supposed to serve that specific purpose, maybe I didn't communicate that well enough.